### DISCUSSION TITLE
Latest Grafana dashboard

### DISCUSSION THREAD

**Discussion by shadz3rg** [moleculerjs/moleculer#1309] [General]
Hello community,
is there official Grafana dashboard for current state framework metrics?
Maybe someone can share?
One what i found is deprecated and not compatible anymore: https://github.com/moleculerjs/moleculer-metrics/blob/master/packages/moleculer-prometheus/grafana-dashboards/Moleculer%20Prometheus.json

**Comment by icebob** [comment]
Use this one: https://github.com/icebob/kantab/blob/master/monitoring/grafana/provisioning/dashboards/Moleculer_Dashboard.json

**Comment by shadz3rg** [comment]
Looking good, thanks!
